lib/tasks/assets.js

```diff
@@ -98,5 +89,16 @@ module.exports = Task.extend({
       stopOnErr: true,
       limit: 2
     });
+  },
+
+  deleteExcluded: function(dir, config){
```

Comment:
@lukemelia Based on how you've invoked this function above, it should be called `deletedExcludedFiles`